core/src/Revolution/Processors/Context/GetList.php

```diff
@@ -40,6 +42,9 @@ class GetList extends GetListProcessor
     /** @var boolean $canCreate Determines whether or not the user can create a context (/duplicate one) */
     public $canCreate = false;
 
+    /** @param boolean $isGridFilter Indicates the target of this list data is a filter field */
+    private $isGridFilter;
```

Comment:
```suggestion
    protected $isGridFilter = false;
```

I think it's better to make this variable protected instead of private, especially since people can extend this class and the variable is used in public methods. But if you have a good argument to make them private, let me know!

Reply:
Good point, I wasn't considering the possible extension of these GetList classes. I'll change that throughout.